notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>It would be good to have the URL itself embedded as a link with a descriptive text. Particularly enough description that someone with a screenreader would know where the link goes. e.g. </p><p><a href="https://www.syfy.com/syfy-wire/most-distant-astronomical-object-ever-seen-in-1962" target="_blank">A SYFY article about 3c273</a> by Phil Plait.</p><p> </p><p>As mentioned, some of these could go at the point in the tutorial where they are relevant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>